### DISCUSSION TITLE
Bromite & Mulch

### DISCUSSION THREAD

**Discussion by rsssxd** [Divested-Mobile/DivestOS-Build#75] [Q&A] [answered]
I couldn't decide between Bromite and Mulch. Are there any security differences between Bromite and Mulch?

Finally, does the Mulch browser have network isolation like Bromite?

**Comment by SkewedZeppelin** [comment, marked as answer]
Mulch tends to update quicker due to fewer patches added and also enables CFI on
all builds.

Mulch also does enable network state partitioning just like Bromite.
Bromite also recently seems to disable the JIT by default.

See:
- https://divestos.org/misc/ch-dates.txt
- https://divestos.org/index.php?page=browsers

**Comment by rsssxd** [comment]
Why is mull browser a system app?

Firefox browsers are weak in terms of security.

I want to use mull browser but not as system app.

**Comment by rsssxd** [comment]
By the way, Mull 99 version is not available in the store.

**Reply by SkewedZeppelin** [reply]
99 is not working right now: https://gitlab.com/relan/fennecbuild/-/merge_requests/13

**Comment by rsssxd** [comment]
Bromite refused to include the latest security update. I will continue to use Mulch again.

https://github.com/bromite/bromite/discussions/2059#discussioncomment-2750269

**Comment by rsssxd** [comment]
I updated mulch and my data disappeared a second time 🥵

**Comment by SkewedZeppelin** [comment]
Mulch tends to update quicker due to fewer patches added and also enables CFI on
all builds.

Mulch also does enable network state partitioning just like Bromite.

See:
- https://divestos.org/misc/ch-dates.txt
- https://divestos.org/index.php?page=browsers

**Comment by SkewedZeppelin** [comment]
Bromite also recently seems to disable the JIT by default.

**Reply by ghost** [reply]
I wish mulch had the easy toggles like disabling Javascript per site alongside the JIT option - that's what makes Bromite stand out.
